### PR TITLE
New markup [do not merge yet]

### PR DESCRIPTION
**Pull request by lfryc**
wip, do not merge yet

### CODE REVIEW

**Comment by cianclarke** [comment]
Didn't get back to this myself today - reviewed so far, looks good, only remark is let's add in bower and squash this down, to remove all that diff :-)

**Comment by lfryc** [comment]
Sounds good, working on that now..

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (cianclarke) [inline bower.json:1]: yay bower! Much nicer

**Comment by cianclarke** [comment]
Looks good to me - LGTM

**Pull request merged**